README.md

```diff
@@ -74,13 +74,14 @@ Astro defines a controller inside of a requirejs module. The module exposes a pr
 
 * `app/app.js` is the entry point of the App
 * `app/global` contains files that provide a common interface for communication between controller objects
-  * `app/global/astro-rpc.js` is an RPC bus that contains registered methods exposed to Astro 
+  * `app/global/app-rpc.js` is an RPC bus that contains registered methods exposed to Astro 
```

Comment:
what prompted this change? I'm fine with it, just wondering the reason behind the name change